documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -537,6 +537,37 @@ Note that these are "resource local" transactions, delegated to the underlying
 Vert.x database client. At present Hibernate Reactive does not integrate with
 container-managed transactions.
 
+== Stateless sessions
+
+An arguably-underappreciated feature of Hibernate is the `StatelessSession`
+interface, which provides a command-oriented, more bare-metal approach to
+interacting with the database.
+
+You may obtain a reactive stateless session from the `SessionFactory`:
+
+[java]
+----
+Stage.StatelessSession ss = getSessionFactory().createStatelessSession();
```

Comment:
I think I'd prefer the `openStatelessSession()` name, as in the original project, as it reminds it's a resource which will need closing to e.g. release connections or other resources (it's not _really_ stateless, is it?)